logistics_project/apps/malawi/warehouse/report_views/reporting_rate.py

```diff
@@ -18,6 +19,9 @@
 class View(warehouse_view.DistrictOnlyView):
 
     def custom_context(self, request):
+        if request.is_facility:
```

Comment:
I'm pretty sure it should always be set so i'm ok failing hard and treating it like a requirement